2024-04-02 18:55:51,672 INFO    MainThread:289285 [wandb_setup.py:_flush():75] Loading settings from /home/presta/.config/wandb/settings
2024-04-02 18:55:51,672 INFO    MainThread:289285 [wandb_setup.py:_flush():75] Loading settings from /scratch/DVC-PyTorch/wandb/settings
2024-04-02 18:55:51,673 INFO    MainThread:289285 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2024-04-02 18:55:51,673 INFO    MainThread:289285 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'main_dvc.py', 'program': 'main_dvc.py'}
2024-04-02 18:55:51,710 INFO    MainThread:289285 [wandb_init.py:_log_setup():386] Logging user logs to /scratch/DVC-PyTorch/wandb/run-20240402_185551-ckalbs82/logs/debug.log
2024-04-02 18:55:51,711 INFO    MainThread:289285 [wandb_init.py:_log_setup():387] Logging internal logs to /scratch/DVC-PyTorch/wandb/run-20240402_185551-ckalbs82/logs/debug-internal.log
2024-04-02 18:55:51,711 INFO    MainThread:289285 [wandb_init.py:init():420] calling init triggers
2024-04-02 18:55:51,711 INFO    MainThread:289285 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {'log': '', 'pretrain': '', 'test': False, 'testhevc': False, 'testvtl': False, 'testmcl': False, 'testauc': False, 'rerank': False, 'allpick': False, 'config': 'DVC_1024.json'}
2024-04-02 18:55:51,711 INFO    MainThread:289285 [wandb_init.py:init():471] starting backend
2024-04-02 18:55:51,711 INFO    MainThread:289285 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-02 18:55:51,737 INFO    MainThread:289285 [backend.py:ensure_launched():219] starting backend process...
2024-04-02 18:55:51,750 INFO    MainThread:289285 [backend.py:ensure_launched():225] started backend process with pid: 289345
2024-04-02 18:55:51,751 INFO    MainThread:289285 [wandb_init.py:init():480] backend started and connected
2024-04-02 18:55:51,766 INFO    MainThread:289285 [wandb_init.py:init():550] updated telemetry
2024-04-02 18:55:51,981 INFO    MainThread:289285 [wandb_init.py:init():581] communicating current version
2024-04-02 18:55:52,765 INFO    MainThread:289285 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.16.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-04-02 18:55:52,766 INFO    MainThread:289285 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2024-04-02 18:55:53,155 INFO    MainThread:289285 [wandb_init.py:init():624] starting run threads in backend
2024-04-02 18:55:54,069 INFO    MainThread:289285 [wandb_run.py:_console_start():1827] atexit reg
2024-04-02 18:55:54,072 INFO    MainThread:289285 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2024-04-02 18:55:54,073 INFO    MainThread:289285 [wandb_run.py:_redirect():1706] Redirecting console.
2024-04-02 18:55:54,075 INFO    MainThread:289285 [wandb_run.py:_redirect():1762] Redirects installed.
2024-04-02 18:55:54,076 INFO    MainThread:289285 [wandb_init.py:init():651] run started, returning control to user process
2024-04-02 18:56:01,025 INFO    MainThread:289285 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 1
2024-04-02 18:56:01,029 INFO    MainThread:289285 [wandb_run.py:_restore():1769] restore
2024-04-02 18:56:03,796 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 737
}

2024-04-02 18:56:03,997 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 737
}

2024-04-02 18:56:04,329 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 737
}

2024-04-02 18:56:04,432 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 5013
}

2024-04-02 18:56:04,535 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 5013
}

2024-04-02 18:56:04,643 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 737
  total_bytes: 5013
}

2024-04-02 18:56:04,745 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5013
  total_bytes: 5013
}

2024-04-02 18:56:04,848 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5013
  total_bytes: 5013
}

2024-04-02 18:56:04,951 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5013
  total_bytes: 5013
}

2024-04-02 18:56:05,054 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5013
  total_bytes: 5013
}

2024-04-02 18:56:05,347 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5013
  total_bytes: 5013
}

2024-04-02 18:56:05,764 INFO    MainThread:289285 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5013
  total_bytes: 5013
}
local_info {
}

2024-04-02 18:56:06,897 INFO    MainThread:289285 [wandb_run.py:_append_files():2194] logging synced files
